tools/scripts/keyboard-abnf-tests/README.md

```diff
@@ -0,0 +1,23 @@
+# Keyboard-abnf-tests
+
+Tests for and against the ABNF files, written in Node.js
+
+## To use
+
+- `npm ci`
+- `npm t`
+
+## To update
+
+Note there are four files. There's a `.d` directory for each ABNF file in keyboards/abnf/.  The "pass" files are expected to pass the ABNF and the "fail" to fail it.  Lines beginning with # are comments and skipped.
+
+- transform-from-required.d/from-match.pass.txt
+- transform-from-required.d/from-match.fail.txt
+- transform-to-required.d/to-replacement.pass.txt
+- transform-to-required.d/to-replacement.fail.txt
```

Comment:
where are these files?

Reply:
They are in that same directory (keyboard-abnf-tests) and were added in #4261

Reply:
that previous PR had a shell script (now deleted) to test the files, this changes it to JS